topics/epigenetics/tutorials/hicexplorer/tutorial.md

```diff
@@ -0,0 +1,259 @@
+
+Matrix correction works in two steps: first a histogram containing the sum of contact per bin (row sum) is produced. This plot needs to be inspected to decide the best threshold for removing bins with lower number of reads. The second steps removes the low scoring bins and does the correction.
+
+> ### {% icon hands_on %} Hands-on: Matrix diagnostic
+> 
+> 1. **hicCorrectMatrix** {% icon tool %}: Run hicCorrectMatrix adjusting the parameters:
+>    - "Range restriction (in bp)" to `Diagnostic plot`
+>    - "Chromosomes to include (and order to plot in)" to `chr2L`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chr2R`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chr3L`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chr3R`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chrX`
+>
+{: .hands_on}
+
+The output of the program prints a threshold suggestion that is usually accurate but is better to revise the histogram plot. The threshold is visualized in the plot as a black vertical line.
+
+In our case the distribution describes the counts per bin of a genomic distance. To remove all bins with a z-score threshold less / more than X means to remove all bins which have less / more counts than X of mean of their specific distribution in units of the standard deviation. Looking at the distribution, we can select the value of -1.6 (lower end) and 1.8 (upper end) to remove. This is given by the –filterThreshold option in hicCorrectMatrix set to 'correct matrix' mode.
+
+> ### {% icon hands_on %} Hands-on: Matrix correction
+>
+> 1. **hicCorrectMatrix** {% icon tool %}: Run hicCorrectMatrix on the output from previous step adjusting the parameters:
+>    - "Range restriction (in bp)" to `Correct matrix plot`
+>    - "Normalize each chromosome separately" to `True`
+>    - "Remove bins of low coverage" to `-1.6`
+>    - "Remove bins of large coverage" to `1.8`
+>    - "Chromosomes to include (and order to plot in)" to `chr2L`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chr2R`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chr3L`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chr3R`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chrX`
+>
+{: .hands_on}
+
+It can happen that the correction stops with:
+
+```
+ERROR:iterative correction:*Error* matrix correction produced extremely large values.
+This is often caused by bins of low counts. Use a more stringent filtering of bins.
+```
+
+This can be solved by a more stringent z-score values for the filter threshold or by a look at the plotted matrix. In our case we see that chromosome Y is having more or less 0 counts in its bins. This chromosome can be excluded from the correction by not defining it for the set of chromosomes that should be corrected (parameter 'Include chromosomes').

```

Comment:
We excluded chr Y already, so maybe reformulate as: `For example chromosome Y is often having more or less 0 counts[...]. Chromosomes like it can be excluded [...]`